### DISCUSSION TITLE
Terminal display error when using oh-my-posh

### DISCUSSION THREAD

**Discussion by KryCool** [JanDeDobbeleer/oh-my-posh#4935] [Troubleshoot]
![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/43307723/db2542fb-b650-44e8-a65c-c34d250d4bd7)

Somebody have same issues please help fix it

**Comment by JanDeDobbeleer** [comment]
You need to install and configure a nerd font for those icons to display, see [here](https://ohmyposh.dev/docs/installation/fonts).
